fix(biome_js_analyze): useConsistentCurlyBraces prserve curly braces … (#6267)
biomejs/biome: .changeset/use_consistent_curly_braces_add_forbidden_chars_check.md

---
"@biomejs/biome": patch
---

> Fixes [`useConsistentCurlyBraces breaks react/no-unescaped-entities rule`](https://github.com/biomejs/biome/issues/5391)
> 
> Adds an additonal check for forbidden characters: ['>', '"', ''', '}'].
> If any of these characters are detected, curly braces will be preserved.

Reviewer: ```suggestion
Fixed [`useConsistentCurlyBraces breaks react/no-unescaped-entities rule`](https://github.com/biomejs/biome/issues/5391)

Added a check for forbidden characters: `>`, `"`, `'` and `}`.
If any of these characters are detected, curly braces will be preserved.
```

Reminder that we have guidelines for the changeset: https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#writing-a-changeset

We use the past
